seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-jdbc/src/main/java/org/apache/seatunnel/flink/jdbc/source/JdbcSource.java

```diff
@@ -273,12 +234,13 @@ private RowTypeInfo getRowTypeInfo() {
         return new RowTypeInfo(typeInformation, names);
     }
 
-    private TypeInformationMap getTypeInformationMap(String driverName) {
-        driverName = driverName.toLowerCase();
-        if (driverName.contains("mysql")) {
+    private TypeInformationMap getTypeInformationMap(String databaseDialect) {
+        if ("mysql".equalsIgnoreCase(databaseDialect)) {
             return new MysqlTypeInformationMap();
-        } else if (driverName.contains("postgresql")) {
+        } else if ("postgresql".equalsIgnoreCase(databaseDialect)) {
             return new PostgresTypeInformationMap();
+        } else if ("oracle".equalsIgnoreCase(databaseDialect)) {
+            return new OracleTypeInformationMap();
```

Comment:
I didn't test on all database, but in my knowledge, the result of `getDatabaseProductName` from oracle driver may not `oracle`, it will contain other character. So it's better to use `StringUtils.containsIgnoreCase(databaseDialect, "xx")`.

Reply:
I checked it in ojdbc source
![image](https://user-images.githubusercontent.com/4036309/167054129-cff47e38-89ff-446c-97ec-fe5f8c30f15c.png)

Reply:
I find an answer in [stackflow](https://stackoverflow.com/questions/6097231/how-to-check-that-we-are-using-oracle-8i-database-in-jdbc), I just worry some corner case.

Reply:
Maybe use equal is fine in here, but use contains will alway more safer.